### PR TITLE
add anirudh-jain profile to community w/ Yash Vardhan <yashvardhan513…

### LINKED ISSUE #1383: [Community] Member Profile: Anirudh Jain
**Current Behaviour**

@anirudhjain75  has been a consistent contributor and community member.

**Desired Situation**
Let's recognizing @anirudhjain75  as a contributor and community member by creating a profile on https://layer5ng.netlify.app/community/members.

- GitHub: anirudhjain75
- Twitter: anirudhjain75
- LinkedIn: anirudh-jain-a19b5a104
- Link to profile picture: https://photos.app.goo.gl/CG5BMERbXxKWB6KN6

---

**Contributor Resources**

The layer5.io website, which uses GatsbyJS and GitHub Pages. Site content is found under the <a href="https://github.com/layer5io/layer5/tree/layer5-ng">`layer5-ng` branch</a>. See <a href="https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md">`layer5` contributing instructions</a>.

### PR DESCRIPTION
**Pull request by Grumpyyash**
…@gmail.com>

Signed-off-by: Yash vardhan <yashvardhan513@gmail.com>

Adding Anirudh-Jain profile to community section 

This PR fixes #1383 of adding the profile of Anirudh-Jain to the community
issue-1383

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/members/anirudh-jain/index.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Anirudh Jain
+position: Contributer
+image_path: ./anirudh-jain.jpg
+github: anirudhjain75
+twitter: anirudhjain75
+linkedin: anirudh-jain-a19b5a104
+blurb: Mesh Head
+status: Active
+badges:
```
Comment (Nikhil-Ladha) [inline src/collections/members/anirudh-jain/index.mdx:10]: Please follow the ordering of times as mentioned in the template (https://github.com/layer5io/layer5/blob/3f738b30b76e0d618910f279572157d5d82738d7/src/collections/members/_member-profile-template/index.mdx).

**Comment by Nikhil-Ladha** [comment]
Also, @anirudhjain75 if you could share a small `bio` and `location` with @Grumpyyash on slack, he might update that as well in this PR.

File: src/collections/members/anirudh-jain/index.mdx (hunk 1 above)
Comment (Nikhil-Ladha) [reply src/collections/members/anirudh-jain/index.mdx:10]: You don't need to add empty fields.

File: src/collections/members/anirudh-jain/index.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+twitter:    anirudhjain75
+linkedin:   anirudh-jain-a19b5a104
+location:   ""
+bio:        ""
+badges:
+  - community
+  - meshary
+blurb:      Mesh Head
+status:     Active
+maintainer: no
```
Comment (Nikhil-Ladha) [inline src/collections/members/anirudh-jain/index.mdx:15]: If it's a `no` it's not required.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Rebase your branch with the upstream/layer5-ng branch to fix the netlify build issue.

**Comment by Grumpyyash** [comment]
I have done the required changes

File: src/collections/members/anirudh-jain/index.mdx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+name:       Anirudh Jain
+position:   Contributer
+image_path: ./anirudh-jain.jpg
+github:     anirudhjain75
+twitter:    anirudhjain75
+linkedin:   anirudh-jain-a19b5a104
+badges:
+  - community
+  - meshary
```
Comment (Nikhil-Ladha) [inline src/collections/members/anirudh-jain/index.mdx:10]: Ahh... last one I guess 😂....typo: meshary-> meshery

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**